src/node_exit_code.h

```diff
@@ -0,0 +1,54 @@
+#ifndef SRC_NODE_EXIT_CODE_H_
+#define SRC_NODE_EXIT_CODE_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+namespace node {
+#define EXIT_CODE_LIST(V)                                                      \
+  V(NoFailure, 0)                                                              \
+  /* 1 was intended for uncaught JS exceptions from the user land but we */    \
+  /* actually use this for all kinds of generic errors. */                     \
+  V(GenericUserError, 1)                                                       \
```

Comment:
I see some of these are using names that are slightly different from the ones used in https://nodejs.org/api/process.html#exit-codes. Should we update the docs to use these names or change the names here? (I'm okay with both sets of names)